iDiff/differs/pipDiff.go

```diff
@@ -0,0 +1,117 @@
+package differs
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"regexp"
+	"strconv"
+
+	"github.com/GoogleCloudPlatform/runtimes-common/iDiff/utils"
+	"github.com/golang/glog"
+)
+
+// layers of two different images.
+func PipDiff(d1file, d2file string) (string, error) {
```

Comment:
There is significant overlap in the differs in terms of having a driver like this which calls a differ specific getPackages function and then gets the diff.  Maybe in a future iteration/refactor we will want to move this repeated code so that it's no longer repeated.